apps/frontend/core-dapp/src/features/deposit/BalanceBox.tsx

```diff
@@ -103,6 +107,31 @@ const BalanceBox: React.FC<{ apr: unknown }> = ({ apr }) => {
             <GreenColor>{aprText}</GreenColor>
           </Value>
         </Row>
+        <Row>
+          <Flex gap={8}>
+            <Icon name="ppo-logo" width="24" height="24" />
+            <div
+              style={{
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+                gap: spacingIncrement(4),
+              }}
+            >
```

Comment:
You could just use the Flex component I think, not sure if it automatically centers everything by default tho

```suggestion
            <Flex
              // may not need alignItems and justifyContent if Flex already centers children
              justifyContent="center"
              alignItems="center"
              gap={4}
            >
```